app/views/case_management/messages/index.html.erb

```diff
@@ -43,7 +43,10 @@
   <% end %>
 <% end %>
 
+
 <% content_for :script do %>
+  <script> window.clientId = <%= @client.id %>;</script>
```

Comment:
Don't know if setting a global JS variable is the best way to do this. Looking for feedback on a better approach.